lib/webdriverio.js

```diff
@@ -200,7 +200,7 @@ let WebdriverIO = function (args, modifier) {
         err.shotTaken = true
         return takeScreenshot(err)
             .catch((e) => logger.log('\tFailed to take screenshot on reject:', e))
-            .then(() => fail(err, onRejected))
+            .then(() => { throw err })
```

Comment:
I am not sure if this is a proper solution for the problem. `fail` does some stuff and can't just be ignored.